resources/partial/workflow-action-springboot/client-extension.yaml

```diff
@@ -0,0 +1,8 @@
+${id}-user-agent:
+  type: oauthApplicationUserAgent
+  scopes:
+    - Liferay.Headless.Admin.Workflow.everything
+
+assemble:
```

Comment:
if they already have assemble, our partials will not merge.  but we can fix `by and by`.

Reply:
agreed!